cleverhans/attacks_tf.py

```diff
@@ -57,6 +63,38 @@ def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
     return adv_x
 
 
+def fgsm_iterative(x, y, model, eps, eps_iter, n_iter,
```

Comment:
this has a different signature than `fgsm`. should we make an Attack class similar to Alex's proposal so that all the attacks have the same signature? (making the attacks standard will probably require breaking the interface to fgsm and incrementing the version number, but I think is worth it to make the attacks standardized in the long run)